js.md

```diff
@@ -22,6 +22,7 @@ sections:
 - "js/analytics.md"
 - "common/data.md"
 - "common/relations.md"
+- "js/schema.md"
```

Comment:
If you're just adding **schema.md** you might want to drop **schemas.md**. In the php section I have it setup to reference schema.md already, might as well be consistent.